scala/scala_maven_import_external.bzl

```diff
@@ -193,13 +196,15 @@ def _serialize_given_rule_import(
         name,
         path,
         srcpath,
+        coordinates,
         attrs,
         props,
         additional_rule_attrs):
     lines = [
         "%s(" % rule_name,
         "    name = %s," % repr(name),
         "    jars = [%s]," % repr(path),
+        "    tags = [\"maven_coordinates=%s\"]," % coordinates,
```

Comment:
Since `coordinates` is non mandatory attribute I think tags should be added conditionally rather than having empty `maven_coordinates=`. wdyt?